src/Hl7.FhirPath.Tests/Tests/BasicFunctionTests.cs

```diff
@@ -189,7 +189,7 @@ public void TestExpressionTodayFunction()
             Assert.AreEqual(P.Date.Today(), scalar("today()"));
 
             // Check greater than
-            isB("today() < @" + P.Date.FromDateTimeOffset(DateTimeOffset.UtcNow.AddDays(1), includeOffset: false));
+            isB("today() < @" + P.Date.FromDateTimeOffset(DateTimeOffset.UtcNow.AddDays(2), includeOffset: false));
```

Comment:
Why is this changed to +2 days instead of 1?

Reply:
This test will otherwise intermittently fail depending on the time of day:  if you are east of GMT and it is already the next day where you are (compared to GMT), this means `today()` is already UTC + 1 day and is not `<`.  Doing `today() <= @.....` would also fix it.

Reply:
(Brian found this bug, who is at GMT+10, so he runs into this much more often than we do)